### PR TITLE
Reactivate Japanese Locale

### PR DESCRIPTION
**Pull request by cassidysymons**
In concert with https://github.com/biocore/microsetta-private-api/pull/542:
- Reactivate Japanese locale
- Update Japanese translations of various front-end content
- Implement external reports in the UI

Notably, some of the content on the results page needed to have the `gettext()` calls adjusted to include dynamic content (the `<span>` tags into which statistics are populated). Due to sentence structure in Japanese being different from English/Spanish, there was no convenient workaround. The end result is that PO files need to be managed very carefully going forward. In a perfect world, it would be nice to find a way to keep text and variables totally separate in a way that allows for translation into all of the languages we support, but it's something we can look at when we scope out the next set of changes on the results page.

### CODE REVIEW

File: microsetta_interface/implementation.py [hunk 1]
```diff
@@ -145,7 +145,7 @@ class Source:
 client_state = RedisCache()
 
 # Countries allowed to view the contents of the My Kits tab
-KITS_TAB_WHITELIST = {'US', 'ES', 'JP'}
+KITS_TAB_WHITELIST = {'US', 'ES', 'JP', 'MX', 'GB'}
```
Comment (wasade) [inline microsetta_interface/implementation.py:148]: no SP?

File: microsetta_interface/templates/account_details.jinja2 [hunk 2]
```diff
@@ -68,6 +68,32 @@
                     $("#state").empty();
                 }
 
+                // Special case to prevent users outside Japan from using Japanese as their language
```
Comment (wasade) [inline microsetta_interface/templates/account_details.jinja2:71]: why?

File: microsetta_interface/implementation.py (hunk 1 above)
Comment (cassidysymons) [reply microsetta_interface/implementation.py:148]: Spain is ES, unless there's another country I'm missing?

File: microsetta_interface/templates/account_details.jinja2 (hunk 2 above)
Comment (cassidysymons) [reply microsetta_interface/templates/account_details.jinja2:71]: We don't have the resources to translate all future changes into Japanese in the future, so we're limiting that locale to Japanese residents in an effort to allow THDMI Japan participants to view their results, while preventing anyone in the US from using the language. We may revisit this decision in the future.

File: microsetta_interface/implementation.py (hunk 1 above)
Comment (wasade) [reply microsetta_interface/implementation.py:148]: lol ya

File: microsetta_interface/templates/account_details.jinja2 (hunk 2 above)
Comment (wasade) [reply microsetta_interface/templates/account_details.jinja2:71]: if the current content works and is correct, why not defer making this special case until a point of deviation in the future?

Comment (cassidysymons) [reply microsetta_interface/templates/account_details.jinja2]: This has been removed from the PR

**Review by wasade — APPROVED** [review]

**Pull request merged**